### PR TITLE
Image name

### PR DESCRIPTION
**Pull request by uhuntu**
Hi there, this PR is an interesting idea to OCR the png/jpg files.

### CODE REVIEW

File: lib/redmine_xapian/patches/attachment_patch.rb
Comment (picman) [inline lib/redmine_xapian/patches/attachment_patch.rb:1]: What is purpose of this change?

File: extra/xapian_indexer.rb
Comment (picman) [inline extra/xapian_indexer.rb:1]: I'd simply replace it width:
```ruby
filespath = Redmine::Configuration['attachments_storage_path'] || Rails.root.join('files')
```

File: lib/redmine_xapian/xapian_search.rb
Comment (picman) [inline lib/redmine_xapian/xapian_search.rb:1]: 1. `Rails.logger.info`... Either remove them before commit or if you think that they are worth for debugging, use `Rails.logger.debug`
2. What is purpose of the change from line 146 down?

**Comment by picman** [comment]
Very well. I have a few comments to your pull request:
1. Do not change code that is not related to the new functionality.
2. Pull request always to the _devel_ branch not _master_
3. If I understand that correctly, in principle you convert all _jpeg_ and _png_ files into _txt_ files and therefore include them into Xapian full-text search. I personally can't imagine any practice usage of such feature but why not. For sure such new functionality should have a paragraph in README.

File: lib/redmine_xapian/patches/attachment_patch.rb
Comment (uhuntu) [reply lib/redmine_xapian/patches/attachment_patch.rb:1]: Sorry for the typos, thanks for your suggestions.

File: extra/xapian_indexer.rb
Comment (uhuntu) [reply extra/xapian_indexer.rb:1]: Yes, yours is better.

File: lib/redmine_xapian/xapian_search.rb
Comment (uhuntu) [reply lib/redmine_xapian/xapian_search.rb:1]: I will remove the debug information.

**Comment by uhuntu** [comment]
> Very well. I have a few comments to your pull request:
> 
> 1. Do not change the code that is not related to the new functionality.
> 2. Pull requests always to the _devel_ branch not _master_
> 3. If I understand that correctly, in principle you convert all _jpeg_ and _png_ files into _txt_ files and therefore include them into Xapian full-text search. I personally can't imagine any practice usage of such feature but why not. For sure such new functionality should have a paragraph in README.

Very well. Thanks for your suggestions:

1. You are right.
2. Will do.
3. Will do.

**Comment by uhuntu** [comment]
Close this as https://github.com/xelkano/redmine_xapian/pull/134 happening.

**Pull request closed without merging**